src/Tintin/Html/Templating.hs

```diff
@@ -152,9 +155,10 @@ tintinHeader info@Project.Info {..} Project.Page {..} =
     meta_ [ name_ "twitter:description"
           , content_ synopsis
           ]
-    meta_ [ name_ "twitter:creator"
-          , content_ githubAuthor
-          ]
+    when (isJust githubAuthor) $  
+      meta_ [ name_ "twitter:creator"
+            , content_ (fromJust $ githubAuthor)
+            ]
```

Comment:
Overall looks great, @J2RGEZ , can you tell if this change breaks the social media cards?

Reply:
This shouldn't be a problem. The only mandatory properties on twitter cards are "card" and "title".

Reply:
Might I suggest:

```haskell
forM_ githubAuthor $ \author ->
    meta_ [ name_ "twitter:creator
          , content_ author
          ]
```

here and later also to avoid the partial usage of `fromJust` ?

Reply:
Curiously enough when I went to try this I got the following error
```{.haskell}
/home/lc/Source/github.com/theam/tintin/src/Tintin/Html/Templating.hs:176:5: error:
    • Do not use 'Foldable' methods on Maybe
      Suggestions:
          Instead of
              for_ :: (Foldable t, Applicative f) => t a -> (a -> f b) -> f ()
          use
              whenJust  :: Applicative f => Maybe a    -> (a -> f ()) -> f ()
              whenRight :: Applicative f => Either l r -> (r -> f ()) -> f ()
      
          Instead of
              fold :: (Foldable t, Monoid m) => t m -> m
          use
              maybeToMonoid :: Monoid m => Maybe m -> m
      
    • In the expression: forM_ githubLink
      In a stmt of a 'do' block:
        forM_ githubLink
          $ \ ghlink -> meta_ [itemprop_ "og:url", content_ githubLink]
      In the second argument of ‘($)’, namely
        ‘do title_ (toHtml $ name <> " - " <> title)
            meta_ [name_ "twitter:card", content_ "summary"]
            meta_
              [name_ "twitter:site",
               content_
                 ("https://s3-eu-west-1.amazonaws.com/worldwideapps/assets/tintin-"
                    <> (Text.toLower $ show color) <> ".png")]
            meta_ [name_ "twitter:title", content_ (name <> " - " <> title)]
            ....’
    |
176 |     forM_ githubLink $ \ghlink ->
```

In any case I'm changing this to whenJust